docs/ruleset_functions/get_energy_required_to_heat_swh_use.md

```diff
@@ -0,0 +1,71 @@
+## get_energy_required_to_heat_swh_use
+
+Description: This function calculates the total energy required to heat the SWH use over the course of a year in btu.  Note - this function does not work for service water heating uses with use_units == "OTHER".  In this case, it will return 0.
+
+Inputs:
+- **swh_use**
+- **RMD**
+
+Returns:
+- **energy_required**: A fixnum indicating the total btu required to heat the water indicated by the swh_use
+
+Function Call:
+
+- get_spaces_served_by_SWH_use
+- get_obj_by_id
+
+Data Lookup: None
+
+Logic:
+
+- get the schedule: `hourly_schedule = swh_use.use_multiplier_schedule`
+- get the temperature at fixture: `supply_temperature = swh_use.distribution_system.design_supply_temperature`
+- get the distribution system: `distribution_system = swh_use.served_by_distribution_system`
```

Comment:
Suggest moving this line up one since you need to use the distribution_system object to get the design_supply_temperature. Right now line 22 does not appear to be referencing the correct data element which woould be .served_by_distribution_system instead of distrubrion_system.